definitions/npm/express_v4.x.x/flow_v0.25.x-/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: any;
```

Comment:
> BTW, if you use json middleware for array, you can receive all values, that are valid json.. So I don't think we have to constrain it to `Object`. So just `Request<T>` without `Request<T:Object>`.

Ok, that's pretty compelling -- I'm onboard with leaving bounds off the typeparam then.
As long as @bassettsj is also onboard here, sounds like we're all in agreement!